app/views/admin/dashboard/_loan_health_message.html.slim

```diff
@@ -0,0 +1,15 @@
+- if status_message[:status] == "healthy"
+  .health-message
+    i.fa.fa-check-circle.ms-tooltip
+    .message.hidden
+      .tooltip-content
+        h4 = status_message[:message]
+- else
+  .health-message
+    i.fa.fa-exclamation-triangle.ms-tooltip
+    .message.hidden
+      .tooltip-content
+        h4 = status_message[:message]
+        ul
+          - status_message[:warnings].each do |warning|
+            li = warning
```

Comment:
There is a fair amount of repeated code in this partial. Could you assign the appropriate icon to a var at the top and then just do a conditional for displaying the `ul` or not?